test/Microsoft.Azure.Devices.Edge.Test.Common/LeafDevice.cs

```diff
@@ -41,8 +42,11 @@ public class LeafDevice
             bool useSecondaryCertificate,
             CertificateAuthority ca,
             IotHub iotHub,
-            CancellationToken token)
+            CancellationToken token,
+            Option<string> modelId)
```

Comment:
Are you able to do `Option<string> modelId = Option.None<string>())` here, so that existing tests that use LeafDevice don't have to change?

Reply:
Unfortunately, no - default params need to be compile-time constant.
I could make it string modelId = null as default then do a null check, but we usually prefer to use Option in our codebase. That being said it is in the test side, so it could be a time to break that rule.

Reply:
Oh yeah...no leave it like this. No big deal.